### ISSUE TITLE
Centralized key bindings

### ISSUE THREAD

**Issue by Yodzorah** [minbrowser/min#417]
What about centralization for all key bindings in the browser? It's not really comfortable to edit key map when it's everywhere. For example, F5 for page reload was assigned directly unlike other bindings and I didn't find some other key bindings like Ctrl+wheel yet, but I suppose they were assigned directly, too. Would be easier to deal with key map located in a single place.

**Comment by PalmerAL** [comment]
This should be fixable once we switch from using IndexedDB to a file for settings.

**Comment by apm** [comment]
I don't see a ticket for moving away from indexedDB, is that in the works?

**Comment by PalmerAL** [comment]
@apm Yeah, I guess I never opened an issue about this. I've created one here: #451. I'm planning to do it eventually, but it hasn't been highest priority, and I haven't had a lot of time to do so.

**Comment by PalmerAL** [comment]
#451 is finally fixed, so I should be able to get to this soon.
